roles/download/defaults/main.yml

```diff
@@ -543,28 +543,39 @@ cni_binary_checksums:
 calicoctl_binary_checksums:
   arm:
     v3.22.1: 0
+    v3.22.2: 0
     v3.21.4: 0
+    v3.21.5: 0
+    
```

Comment:
We don't need to put a blank line in the same block for `arm`.

Reply:
Removed and sorry about that.